examples/service/Makefile

```diff
@@ -31,7 +31,7 @@ client: ## Runs the uncompiled example client code
 custom-grpc-client: ## Runs the uncompiled example custom grpc client code
 	dapr run --app-id custom-grpc-client \
 		 -d ./config \
-		 --dapr-http-max-request-size 41 \
+		 --max-body-size 4Mi \
```

Comment:
Looks like the sidecar was terminating following the updated arg `--max-body-size`. This change should be fine... the validation workflow was pinned to a previous dapr runtime version (1.14.1) but this shouldn't have made a difference, it looks like the CLI hasn't implemented the required changes to accommodate the deprecation of `--dapr-http-max-request-size`

My recommendation is to revert this change until the CLI has implemented this.